packages/fxa-settings/src/pages/ConfirmSignin/index.tsx

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { usePageViewEvent } from '../../lib/metrics';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { RouteComponentProps } from '@reach/router';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+
+export type ConfirmSigninProps = {
+  canGoBack?: boolean;
+  email: string;
+  goBackCallback?: Function;
+  isOpenWebmailButtonVisible: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
+};
+
+export type WebmailValues = {
+  buttonText: string;
+  link: string;
+};
+
+const ConfirmSignin = ({
+  canGoBack,
+  email,
+  goBackCallback,
+  isOpenWebmailButtonVisible,
+}: RouteComponentProps & ConfirmSigninProps) => {
+  usePageViewEvent('confirm-signin', {
```

Comment:
Small reminder to add the metrics event to the [spreadsheet](https://docs.google.com/spreadsheets/d/1na5BqUNefrgsK-7wTlKoy-GuacLZ0rfSZ1bLiJkwFOk/edit#gid=0) :)